Documents/MobileAppActors.txt

```diff
@@ -0,0 +1,17 @@
+Actors: Regular User, Artist, Admin
+
+Regular User - Draw on Digital Prosthetic, Browse Gallery, Contact Artists
```

Comment:
**Go to drawing screen**- how to get to the screen? what will the user see?
After you appear on the screen, you can:
**Select a brush**- what is the brush selection? Maybe there is only one default brush. What is it? What is a default brush? How do different brushes affect the canvas?
**Undo**- can you undo your drawing?
**Eraser**- is there an eraser function?
**Select a canvas**- Does canvas selection happen before or after getting to the drawing screen?